SolveMeFirst.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+class Solution {
+
+    static int solveMeFirst(int a, int b) { 
+      int c=a+b;
```

Comment:
No need to declare an extra variable. you can return directlya+b. one extra line of code will mean extra maintenance. Please keep this in mind

Reply:
@narayanagunturi I have resolved this problem and will keep in mind next time.